### PR TITLE
feat: RDCC-1287 added validation for blank values

### PR DESCRIPTION
**Pull request by choudsc**
JIRA link (if applicable)

https://tools.hmcts.net/jira/browse/RDCC-1220

Change description

 added validation for blank values

Does this PR introduce a breaking change? (check one with "x")
No

### CODE REVIEW

**Comment by choudsc** [comment]
reopening

File: charts/rd-judicial-data-load/values.preview.template.yaml [hunk 1]
```diff
@@ -18,8 +18,24 @@ java:
...
+      key: primaryBlobServiceEndPoint
+blobstorage:
+  resourceGroup: rd
+  teamName: "RD"
+  location: uksouth
+  setup:
+    containers:
+    - jud-ref-data
+    - jud-ref-data-archive
+  enabled: true
```
Comment (shashank-rastogi) [inline charts/rd-judicial-data-load/values.preview.template.yaml:41]: why blob store chart changes are part of your pr?

Comment (shashank-rastogi) [reply charts/rd-judicial-data-load/values.preview.template.yaml:41]: This PR should have changes for  adding validation for blank values only

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -79,8 +79,8 @@ spring:
 
 # Scheduler configuration
 scheduler:
-  camel-route-config: 0 0 22 * * ?
-  camel-leaf-router-config: 0 0 21 * * ?
+  camel-route-config: 0 0/7 * * * ?
+  camel-leaf-router-config: 0 0/5 * * * ?
```
Comment (shashank-rastogi) [inline src/main/resources/application.yaml:84]: correct the scheduler timings

**Pull request closed without merging**